R/build-home.R

```diff
@@ -15,68 +15,68 @@
 #' iterating when experimenting with site styles.
 #'
 #' # Home page
-#' 
+#'
 #' The main content of the home page (`index.html`) is generated from
 #' `pkgdown/index.md`, `index.md`, or `README.md`, in that order.
-#' Most packages will use `README.md` because that's also displayed by GitHub 
-#' and CRAN. Use `index.md` if you want your package website to look 
-#' different to your README, and use `pkgdown/index.md` if you don't want that 
+#' Most packages will use `README.md` because that's also displayed by GitHub
```

Comment:
Can you figure out what setting is trimming these trailing whitespaces and either turn it off or apply it to every file in one PR? Otherwise it makes it a bit hard for me to quickly scan and find the main differences.

Reply:
It is in .Rproj setting, when I save a file, it trims spaces... I don't know why this didn't happen for other contributors.

![image](https://github.com/r-lib/pkgdown/assets/52606734/2051a6bc-c409-465c-8fe7-17e366bdfcea)

Possibly relates to https://github.com/rstudio/rstudio/issues/9902

Reply:
I'd recommend merging #2566 first, so that I can merge and I can take a last look at this PR before merge.

Reply:
Oh yeah, it's probable that I have this turned off because it tends to cause problems when I drop into some random repo to make a tactical fix.

Reply:
I will revisit later with fresh eyes this PR with your comments in mind and will let you know when it is ready for final review!